src/app/mapControls/FiltersContainer.js

```diff
@@ -0,0 +1,88 @@
+define([
+    'app/config',
+    'app/mapControls/Filter',
+
+    'dijit/_TemplatedMixin',
+    'dijit/_WidgetBase',
+    'dijit/_WidgetsInTemplateMixin',
+
+    'dojo/_base/declare',
+    'dojo/_base/lang',
+    'dojo/dom-construct',
+    'dojo/text!app/mapControls/templates/FiltersContainer.html',
+    'dojo/topic'
+], function (
+    config,
+    Filter,
+
+    _TemplatedMixin,
+    _WidgetBase,
+    _WidgetsInTemplateMixin,
+
+    declare,
+    lang,
+    domConstruct,
+    template,
+    topic
+) {
+    return declare([_WidgetBase, _TemplatedMixin, _WidgetsInTemplateMixin], {
+        // description:
+        //      Holds filter widgets and coordinates their work
+        templateString: template,
+        baseClass: 'filters-container',
+        widgetsInTemplate: true,
+
+        // filters: _Filter[]
+        //      list of filters for this widget
+        filters: null,
+
+        // Properties to be sent into constructor
+
+        postCreate: function () {
+            // summary:
+            //      Overrides method of same name in dijit._Widget.
+            console.log('app.mapControls.FiltersContainer::postCreate', arguments);
+
+            this.filters = [
+                new Filter({
+                    name: 'Project Status',
+                    items: config.lut.projectStatus.map(function (item) {
+                        return [item, item];
+                    }),
+                    fieldName: config.fieldNames.Status,
+                    fieldType: Filter.TYPE_TEXT
+                }, domConstruct.create('div', null, this.container)),
+                new Filter({
+                    name: 'Feature Type',
+                    items: config.lut.featureType,
+                    fieldName: config.fieldNames.TypeCode,
+                    fieldType: Filter.TYPE_NUMBER,
+                    relatedTableQuery: true,
+                    anyAllToggle: true
+                }, domConstruct.create('div', null, this.container))
+            ];
+            this.filters.forEach(function (f) {
```

Comment:
I'm happy with the way it is, but for your professional development purposes

> The idea behind event delegation is that instead of attaching a listener to an event on each individual node of interest, you attach a single listener to a node at a higher level, which will check the target of events it catches to see whether they bubbled from an actual node of interest; if so, the handler's logic will be performed.